example/index.html

```diff
@@ -16,7 +16,6 @@
     <script src="https://requirejs.org/docs/release/2.3.6/minified/require.js"></script>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/prism/1.27.0/prism.min.js"></script>
     <script src="shared.js"></script>
-
```

Comment:
This is just a whitespace change, did you want to track this here?

Reply:
Naw, I'll remove this. Thanks.